include/api.h

```diff
@@ -41,8 +41,8 @@
 
 #define PROS_VERSION_MAJOR 3
 #define PROS_VERSION_MINOR 2
-#define PROS_VERSION_PATCH 0
-#define PROS_VERSION_STRING "3.2.0"
+#define PROS_VERSION_PATCH 1
+#define PROS_VERSION_STRING "3.2.1-dirty.39.b5613ca"
```

Comment:
This and Version should be reverted.

Reply:
:+1: Will do

Reply:
Can I build the kernel myself or is that still not a thing ?

Reply:
Not out of the box due to restrictions from VEX on the sdk (You need to sign an NDA to get access to it). You could probably grab the headers you need from [here](https://api.vexcode.cloud/v5/html/dir_a74dadfb59ed28ed8028dab2928701b9.html) though, and if you put them in include that'll probably get you far enough.